test/data/3.2/foreman_api.json

```diff
@@ -0,0 +1 @@
p\u003e\n","required":false,"allow_nil":true,"allow_blank":false,"validator":"Must be a Hash","expected_type":"hash","metadata":null,"show":true,"validations":[]}]}],"returns":[],"examples":["PUT /api/hosts/host70/interfaces/12\n{\n  \"interface\": {\n    \"name\": \"hsobarwiep\"\n  }\n}\n200\n{\n  \"subnet_id\": null,\n  \"subnet_name\": null,\n  \"subnet6_id\": null,\n  \"subnet6_name\": null,\n  \"domain_id\": null,\n  \"domain_name\": null,\n  \"created_at\": \"2021-05-18 11:45:36 UTC\",\n  \"updated_at\": \"2021-05-18 11:45:36 UTC\",\n  \"managed\": true,\n  \"identifier\": \"eth35\",\n  \"id\": 12,\n  \"name\": \"hsobarwiep\",\n  \"ip\": \"0.0.0.17\",\n  \"ip6\": null,\n  \"mac\": \"00:33:45:ab:00:16\",\n  \"mtu\": null,\n  \"fqdn\": \"hsobarwiep\",\n  \"primary\": false,\n  \"provision\": false,\n  \"type\": \"interface\",\n  \"virtual\": false\n}"],"metadata":null,"see":[],"headers":[],"show":true},{"doc_url":"../apidoc/v2/interfaces/destroy","name":"destroy","apis":[{"api_url":"/api/hosts/:host_id/interfaces/:id","http_method":"DELETE","short_description":"Delete a host's interface","deprecated":null}],"formats":null,"full_description":"","errors":[],"params":[{"name":"location_id","full_name":"location_id","description":"\n\u003cp\u003eSet the current location context for the request\u003c/p\u003e\n","required":false,"allow_nil":false,"allow_blank":false,"validator":"Must be a Integer","expected_type":"numeric","metadata":null,"show":true,"validations":[]},{"name":"organization_id","full_name":"organization_id","description":"\n\u003cp\u003eSet the current organization context for the request\u003c/p\u003e\n","required":false,"allow_nil":false,"allow_blank":false,"validator":"Must be a Integer","expected_type":"numeric","metadata":null,"show":true,"validations":[]},{"name":"host_id","full_name":"host_id","description":"\n\u003cp\u003eID or name of host\u003c/p\u003e\n","required":true,"allow_nil":false,"allow_blank":false,"validator":"Must be a String","expected_type":"string","metadata":null,"show":true,"validations":[]},{"name":"id","full_name":"id","description":"\n\u003cp\u003eID of interface\u003c/p\u003e\n","required":true,"allow_nil":false,"allow_blank":false,"validator":"Must be a String","expected_type":"string","metadata":null,"show":true,"validations":[]}],"returns":[],"examples":["DELETE /api/hosts/host66/interfaces/2\n{\n  \"interface\": {}\n}\n200\n{\n  \"id\": 2,\n  \"mac\": \"00:33:45:ab:00:10\",\n  \"ip\": \"0.0.0.11\",\n  \"name\": null,\n  \"host_id\": 1,\n  \"subnet_id\": null,\n  \"domain_id\": null,\n  \"attrs\": {},\n  \"created_at\": \"2021-05-18T11:45:35.580Z\",\n  \"updated_at\": \"2021-05-18T11:45:35.580Z\",\n  \"provider\": null,\n  \"username\": null,\n  \"password\": null,\n  \"virtual\": false,\n  \"link\": true,\n  \"identifier\": \"eth28\",\n  \"tag\": \"\",\n  \"attached_to\": \"\",\n  \"managed\": true,\n  \"mode\": \"balance-rr\",\n  \"attached_devices\": \"\",\n  \"bond_options\": \"\",\n  \"primary\
```

Comment:
Re-generate please, otherwise tests will fail.

Reply:
I regenerate but I don't think it works...